processing_scripts/New_Images_From_ROIs.py

```diff
@@ -4,6 +4,9 @@
 images with the regions within the ROIs, and saves them back to the server.
 
 This script is adapted from [SCRIPTS]/omero/util_scripts/Images_From_ROIs.py
+This is an adapted version of Alex Herbert's script so it can be used
```

Comment:
I'd move this new info to below 'The script has been modified to allow...' info - it's a bit confusing to put it here after the mention of the script Alex originally adapted it from.